arbnode/node.go

```diff
@@ -795,6 +802,15 @@ func createNodeImpl(
 		return nil, err
 	}
 
+	var bpVerifier *contracts.BatchPosterVerifier
+	if l1client != nil {
+		seqInboxCaller, err := bridgegen.NewSequencerInboxCaller(sequencerInboxAddr, l1client)
+		if err != nil {
+			return nil, err
+		}
+		bpVerifier = contracts.NewBatchPosterVerifier(seqInboxCaller)
+	}
+	sigVerifier := signature.NewVerifier(false, nil, bpVerifier)
```

Comment:
`requireSignature` seems to be hardcoded to false here, shouldn't it be configurable?

Reply:
:+1: